plugins/topology/src/utils/transform-utils.ts

```diff
@@ -0,0 +1,266 @@
+import {
+  EdgeModel,
+  Model,
+  NodeModel,
+  NodeShape,
+} from '@patternfly/react-topology';
+import * as _ from 'lodash';
+import {
+  GROUP_HEIGHT,
+  GROUP_PADDING,
+  GROUP_WIDTH,
+  NODE_HEIGHT,
+  NODE_PADDING,
+  NODE_WIDTH,
+  TYPE_APPLICATION_GROUP,
+  TYPE_CONNECTS_TO,
+} from '../const';
+import {
+  OdcNodeModel,
+  TopologyDataModelDepicted,
+} from '../types/topology-types';
+import { K8sResourceKind } from '../types/types';
+import { groupVersionFor } from './topology-utils';
+
+export type ConnectsToData = { apiVersion: string; kind: string; name: string };
+
+export const WorkloadModelProps = {
+  width: NODE_WIDTH,
+  height: NODE_HEIGHT,
+  group: false,
+  visible: true,
+  style: {
+    padding: NODE_PADDING,
+  },
+};
+
+/**
+ * create node data for graphs
+ */
+export const getTopologyNodeItem = (
+  resource: K8sResourceKind,
+  type: string,
+  data: any,
+  nodeProps?: Omit<OdcNodeModel, 'type' | 'data' | 'children' | 'id' | 'label'>,
+  children?: string[],
+  resourceKind?: string,
+  shape?: NodeShape,
+): OdcNodeModel => {
+  const uid = resource.metadata?.uid;
+  const name = resource.metadata?.name;
+  const label = resource.metadata?.labels?.['app.openshift.io/instance'];
```

Comment:
Shouldn't this be

```suggestion
  const label = resource.metadata?.labels?.['app.kubernetes.io/instance'];
```

At least that's what is used at [`utils/topology-utils.ts`](https://github.com/janus-idp/backstage-plugins/pull/104/files#diff-8eed6f305ecf78ad964f1776378d9f7c65b816225105701acbf82ff4773cc8a2R96) above.

Maybe the `app.kubernetes.io/instance` should be a proper global constant.